frontend/components/form/Field.vue

```diff
@@ -17,13 +17,18 @@ const props = withDefaults(defineProps<Props>(), {
   required: false,
   loading: false,
   completed: false,
-  requiredLabel: 'Required',
+  requiredLabel: 'Needs Attention',
   loadingLabel: 'Loading...',
   completedLabel: '',
 })
 const slots = useSlots()
 
 const helpTextExists = computed(() => props.helpText !== '' || slots['help-text'] !== undefined)
+const invalidFieldName = computed(() => {
+  if (props.required && !props.completed) { return props.requiredLabel }
+  return undefined
+})
+defineExpose({ invalidFieldName })
```

Comment:
I don't actually see this used anywhere, but in either case, here's an example of where I think we can get away without using an `expose`. Instead, I'd run your `invalidFieldName` logic on an event like `blur` and, if invalid, emit an `@invalidField` or whatever event. That way it's a "push" versus a "pull".

Reply:
Ah yes! It's no longer nescessary - it was the anti-pattern I was trying to design out. I'll leave this open to discuss the other part of the suggesiton.